locale/fi/LC_MESSAGES/django.po

```diff
@@ -23,17 +23,17 @@ msgstr "Tämä kenttä on pakollinen."
 
 #: templates/_pagination.html
 #, python-format
-msgid "%(count)s student message found"
-msgid_plural "%(count)s student messages found"
-msgstr[0] "%(count)s opiskelijan palaute löytyi"
-msgstr[1] "%(count)s opiskelijoiden palautetta löytyi"
+msgid "%(count)s conversation found with"
+msgid_plural "%(count)s conversations found with"
+msgstr[0] "Löytyi %(count)s keskustelu ja siinä"
+msgstr[1] "Löytyi %(count)s keskustelua ja niissä"
 
 #: templates/_pagination.html
 #, python-format
-msgid "in %(count)s conversation."
-msgid_plural "in %(count)s conversations."
-msgstr[0] "%(count)s keskustelusta."
-msgstr[1] "%(count)s keskustelusta."
+msgid "%(count)s matching student message."
+msgid_plural "%(count)s matching student messages."
+msgstr[0] "%(count)s hakuehtoihin osuva opiskelijan viesti."
+msgstr[1] "%(count)s hakuehtoihin osuvaa opiskelijan viestiä."
```

Comment:
Is `%(count)s hakuehdot täyttävä opiskelijan viesti.` and `%(count)s hakuehdot täyttävää opiskelijan viestiä.` better?

Reply:
It probably would be, I copied directly what @JSorva had proposed in the issue, but he was fine with your suggestion as well. And he also suggested that "Löytyi" could be removed from the beginning to further emphasize the number. (If it's added to the end in Finnish, it would still work as a sentence.

Reply:
Hmm, or does "X keskustelua ja niissä Y hakuehdot täyttävää opiskelijan viestiä löytyi." sound too "kapulakielinen", would it be better to be e.g. "X keskustelua löytyi, joissa on Y hakuehdot täyttävää opiskelijan viestiä."

Reply:
yes, that's better